cmd/fscrypt/prompt.go

```diff
@@ -282,7 +282,7 @@ func promptForProtector(options []*actions.ProtectorOption) (int, error) {
 	}
 
 	if numLoadErrors > 0 {
-		fmt.Print(wrapText("NOTE: %d of the %d protectors failed to load. "+loadHelpText, 0))
+		fmt.Print(wrapText(fmt.Sprintf("NOTE: %d of the %d protectors failed to load. ", numLoadErrors, numOptions)+loadHelpText, 0) + "\n")
```

Comment:
This long line is difficult to read.  Can you break it up, like the following:

                warning := fmt.Sprintf("NOTE: %d of the %d protectors failed to load. ", numLoadErrors, numOptions)
                fmt.Print(wrapText(warning+loadHelpText, 0) + "\n")